src/android/CodePush.java

```diff
@@ -109,17 +113,17 @@ private boolean execUpdateSuccess(CallbackContext callbackContext) {
             /* save reporting status for first install */
             try {
                 String appVersion = Utilities.getAppVersionName(cordova.getActivity());
-                codePushReportingManager.reportStatus(CodePushReportingManager.Status.STORE_VERSION, null, appVersion, mainWebView.getPreferences().getString(DEPLOYMENT_KEY_PREFERENCE, null), this.mainWebView);
+                codePushReportingManager.reportStatus(new StatusReport(ReportingStatus.STORE_VERSION, null, appVersion, mainWebView.getPreferences().getString(DEPLOYMENT_KEY_PREFERENCE, null)), this.mainWebView);
```

Comment:
I think I'll rename it to `execNotifyApplicationReady`. That makes JS and native use the same name which I think is way simpler. The RN implementation does that too.